source/features/raw-file-link.tsx

```diff
@@ -10,17 +10,16 @@ const createRawUrl = (href: string): string => {
 };
 
 function addRawButtons(): void {
-	for (const anchor of select.all<HTMLAnchorElement>('.file-header .file-actions > div.BtnGroup [href*=\'/blob/\']:last-child')) {
+	for (const anchor of select.all<HTMLAnchorElement>('.file-header .file-actions > div > details [href*=\'/blob/\']')) {
```

Comment:
This can be `.file .js-file-header-dropdown [href*=\'/blob/\']`. You can even drop the `.file` part at the start too.

Reply:
Hey @notlmn , what's the benefit of that selector over others?

Reply:
> what's the benefit of that selector over others?

The shorter and less specific the better. Many `>` will make it too dependent on small variations

Reply:
Also perhaps we should avoid that `/blob/` check because that would match any link in a `/blob/` repo or `/blob/` directory. Let's try something like `[data-ga-click^="View file"]`